Umbraco-Cloud/Set-Up/Manage-CDN-Caching/index.md

```diff
@@ -0,0 +1,128 @@
+
+## Caching Explained
+
+When Caching is enabled on your project it means that static assets like javascript, css and images are going to be cached in the Content Delivery Network (CDN) used by Umbraco Cloud. How assets are cached are very much up to you, as you can control it through 'cache control headers'. By default Umbraco Cloud will enforce a minimum Time to Live (TTL) based on the Plan of your Umbraco Cloud Project, which means that assets cannot be cached for a short period of time then what your Plan allows. You can always choose a longer duration - especially if you don't expect your assets to change.
+
+These are the types of files that are cached as static assets through the CDN: '7z', 'csv', 'gif', 'midi', 'png', 'tif', 'zip', 'avi', 'doc', 'gz', 'mkv', 'ppt', 'tiff', 'zst', 'avif', 'docx', 'ico', 'mp3', 'pptx', 'ttf', 'apk', 'dmg', 'iso', 'mp4', 'ps', 'webm', 'bin', 'ejs', 'jar', 'ogg', 'rar', 'webp', 'bmp', 'eot', 'jpg', 'otf', 'svg', 'woff', 'bz2', 'eps', 'jpeg', 'pdf', 'svgz', 'woff2', 'class', 'exe', 'js', 'pict', 'swf', 'xls', 'css', 'flac', 'mid', 'pls', 'tar', 'xlsx'.
+
+If you want to disable caching on certain types of static assets, you can use a 'no-cache' cache control header, which will be respected by the caching strategy in the CDN. You can utilize an outbound rewrite rule in order to add such a cache control header to the request. 
+
+The following example adds a cache-control header with 'no-cache' as the value when the requested Url contains a PDF file:
+
+```
+<rewrite>
+    <outboundRules>
+        <rule name="Set Cache-Control - No-Cache PDF">
+            <match serverVariable="RESPONSE_Cache_Control" pattern=".*" />
+            <conditions>
+                <add input="{REQUEST_URI}" pattern="\.(pdf)$" />
+            </conditions>
+            <action type="Rewrite" value="no-cache"/>
+        </rule>
+    </outboundRules>
+</rewrite>
+```
+
+Please note that the webpage itself is not cached when CDN Caching is enabled.
+
+## Cache Everything Explained
+
+When Cache Everything is enabled it means that everything including the webpage is cached in the CDN. So in addition to static assets the webpage will also be cached and served from the CDN instead of loaded from the origin. When a webpage is cached it will be stripped of any cookies that are otherwise part of the request, so if you are utilizing cookies as part of the website please be aware of the implications of using Cache Everything.
+
+When using Cache Everything you should use a Cache TTL, which matches the Editors expectations of when the webpage is refreshed with a new version loaded from the origin. As an example, chosing a Cache TTL of 2 hours means that the webpage will be served from the cache for 2 hours and then it will be refreshed with a copy from the origin. If Editors makes changes every 30 minutes then they will have to wait at least two hours until they can see the changes on the website.
```

Comment:
```suggestion
When Cache Everything is enabled it means that everything including the webpage is cached in the CDN. So in addition to static assets, the webpage will also be cached and served from the CDN instead of loaded from the origin. When a webpage is cached it will be stripped of any cookies that are otherwise part of the request, so if you are utilizing cookies as part of the website please be aware of the implications of using Cache Everything.

When using Cache Everything you should use a Cache TTL, which matches the Editor's expectations of when the webpage is refreshed with a new version loaded from the origin. As an example, choosing a Cache TTL of 2 hours means that the webpage will be served from the cache for 2 hours and then it will be refreshed with a copy from the origin. If Editors make changes every 30 minutes then they will have to wait at least two hours until they can see the changes on the website.
```